src/Commands/Stats.ts

```diff
@@ -0,0 +1,90 @@
+import { Command } from '../Infrastructure/System/Command';
+import { Message, Guild, GuildMember, MessageEmbed, Role, Collection } from 'discord.js';
+import moment = require('moment');
+
+/**
+ * ## Stats
+ * Display user Discord stats.
+ *
+ * Usage:
+ * `.stats`
+ * `.stats <discord ID/mention>`
+ *
+ * @category Commands: Misc
+ */
+export default class Stats extends Command {
+	public constructor() {
+		super({
+			name: 'Discord Stats',
+			command: [ 'stats' ],
+			description: 'Display user Discord stats.',
+			argumentDescriptions: [],
+			usage: '<prefix>stats <Argument>?',
+			roles: [],
+			commandsPerMinute: 5,
+			commandSurgeMax: 2.0,
+		});
+	}
+
+	public async execute(message: Message, args: string[]): Promise<void> {
+		const guild: Guild | undefined = this.client.guilds.cache.get('529310206639996928');
+		let guildMember: GuildMember | undefined | null = null;
+
+		if (guild && args.length > 0) {
+			if (guild.members.fetch(args[0])) {
+				guildMember = await guild.members.fetch(args[0]);
+			}
+		} else if (guild) {
```

Comment:
We should probably just put a return at the top if guild is null/undefined, that way we don't have to check if guild is not null in so many different places.